client/settings/payment-request-section/index.js

```diff
@@ -219,6 +226,67 @@ const PaymentRequestSection = () => {
 							</div>
 						</li>
 					) }
+					{ displayExpressPaymentMethods && (
```

Comment:
Hey @wjrosa, I noticed that according to [this documentation](https://docs.stripe.com/elements/express-checkout-element/accept-a-payment#supported-payment-methods), it seems that Amazon Pay does not require the `card` payment method type so we don't need this check here. Would you prefer that I open another follow-up PR or address it in this PR?

Reply:
Feel free to address this here 👍 . I can take another look when you do if the change is considerable

Reply:
@wjrosa updated here [ed94498](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3740/commits/ed9449818e38d1c77171c8ec0fbb119bd9deea24). Thank you!